ccgains/historic_data.py

```diff
@@ -415,3 +415,232 @@ def prepare_request(self, dtime):
 
             store.put(key, self.data, format="fixed")
             return self.data
+
+
+class HistoricDataAPICoinbase(HistoricData):
+    last_query_time = pd.Timestamp.now()
+
+    def __init__(self, cache_folder, unit, interval='H'):
+        """Initialize a HistoricData object which tranparently fetches data
```

Comment:
"... which will *tran**s**parently* fetch ..." No need to repeat my stupid mistakes. :-p

Reply:
heh, yup, copy-paste got me there :)